src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -180,7 +181,13 @@ def get_or_create_user_by_email(self, *, email: str) -> RpcUser:
                     name=email,
                 )
             else:
-                user = User.objects.get(email=email)
+                user_query = User.objects.filter(email=email)
+                # Users are not supposed to have the same email but right now our auth pipeline let this happen
+                # So let's not break the user experience
+                if user_query.count() > 1:
+                    logger = logging.getLogger("user:provisioning")
+                    logger.error(f"email {email} has more than 1 user")
```

Comment:
`logger.error` will create a sentry issue, is this intended?
`logger.info` will log without creating anything

Reply:
Yes, we want issue, we don't want exception.